resources/ext.wikibase.facetedsearch.js

```diff
@@ -84,8 +84,12 @@ function onInstancesClick( event, instanceId ) {
  * @param {?string} mode
  */
 function onListFacetInput( facet, propertyId, mode ) {
+	const selectedValues = getListFacetSelectedValues( facet );
+	if ( selectedValues.length === 0 ) {
+		return;
+	}
```

Comment:
@alistair3149 I realise the error here now (https://github.com/ProfessionalWiki/WikibaseFacetedSearch/issues/172). When deselecting everything, this function will return and not update the search.